pkg/istio-agent/sds-agent.go

```diff
@@ -185,11 +185,26 @@ func NewSDSAgent(discAddr string, tlsRequired bool, pilotCertProvider, jwtPath,
 
 	discHost, discPort, err := net.SplitHostPort(discAddr)
 	if err != nil {
-		log.Fatalf("Invalid discovery address (%v): %v", discAddr, err)
+		log.Fatala("Invalid discovery address", discAddr, err)
```

Comment:
why are you changing this? The Loga doesn't add spaces so it's hard to read

Reply:
Probably merge artifacts. Why doesn't it add spaces, standard go log does - is it our wrapper?
Will fix it.

Reply:
fixed